pandas/tslib.pyx

```diff
@@ -707,12 +708,26 @@ NaT = NaTType()
 
 iNaT = util.get_nat()
 
-
 cdef inline bint _checknull_with_nat(object val):
     """ utility to check if a value is a nat or not """
     return val is None or (
         PyFloat_Check(val) and val != val) or val is NaT
 
+cdef inline bint _check_all_nulls(object val):
+    """ utility to check if a value is any type of null """
+    cdef bint res = 0
+    if PyFloat_Check(val):
```

Comment:
shouldn't you just return at each stage? rather than setting a value then returning it at the end